Zc-specification/Zc.adoc

```diff
@@ -120,6 +121,22 @@ Therefore common ISA strings can be updated as follows to include the relevant Z
 * RV32IMC becomes RV32IM_Zce
 * RV32IMCF becomes RV32IMF_Zce
 
+[#misaC]
+=== MISA.C
+
+If all the instructions from the C extension are present, and the MISA CSR, has been implemented, then MISA.C is set. This happens if the following extensions are selected:
+
+* Zca and not F
+* Zca, Zcf and F is specified (RV32 only)
+* Zca, Zcf and Zcd if D is specified (RV32 only)
+** this configuration excludes Zcmp, Zcmt
+* Zca, Zcd if D is specified (RV64 only)
+** this configuration excludes Zcmp, Zcmt
+
+Therefore MISA.C can be set without directly specifying the C extension.
+
+Specifying Zce sets MISA.C if D is not implemented. If D _is_ implemented then Zce does _not_ set MISA.C as it cannot include Zcd because some of the encodings have been repurposed by Zcmp and Zcmt.

```

Comment:
Similar point here.  "Specified" isn't an ISA concept, so you can just nuke this.  The rules you wrote at https://github.com/riscv/riscv-code-size-reduction/pull/230/files#diff-9d4bb14a2f0a03bd0cc4e07c81f50a143f47d14af38bf10a4ed97fd449b7bd97R129-R134 are sufficient to describe the semantics.